docs/source/es/quicktour.mdx

```diff
@@ -87,7 +87,7 @@ El pipeline descarga y almacena en caché el [modelo preentrenado](https://huggi
 
 ```py
 >>> clasificador("Estamos muy felices de mostrarte la biblioteca de 🤗 Transformers.")
-[{'label': 'POS', 'score': 0.9916}]
+[{'label': 'POS', 'score': 0.9320}]
```

Comment:
The used reop. `pysentimiento/robertuito-sentiment-analysis` has some recent updates on modeling/config/tokenizer files.